src/openrct2/localisation/StringIds.h

```diff
@@ -67,20 +67,17 @@ enum : StringId
     STR_MONTH_SHORT_NOV = 820, // Unused
     STR_MONTH_SHORT_DEC = 821, // Unused
 
-    STR_CLOSE_X = 824,
     STR_CHOSEN_NAME_IN_USE_ALREADY = 825,
     STR_TOO_MANY_NAMES_DEFINED = 826,
     STR_NOT_ENOUGH_CASH_REQUIRES = 827,
 
-    STR_ABOUT = 847,
-
     // STR_0858 to 0864 are removed.
     STR_STRINGID = 865,
     STR_ARG_2_STRINGID = 866, // Unused
     STR_ARG_4_STRINGID = 867, // Unused
     STR_ARG_8_STRINGID = 869, // Unused
```

Comment:
Any reason to keep these around? There are many more ofcourse, just wondering now and may be better left untouched for another PR.

Reply:
I'm not quite sure, to be honest. We've deleted plenty of unused strings in the past, after all. Looks like they've been marked as such for a while, because the lines were last touched in 61d38511bce (2017), and that was just formatting...